09-Multiparticle-Collision-Dynamics/01-Pressure-Driven-Flow.ipynb

```diff
@@ -0,0 +1,505 @@
+{
```

Comment:
<p>Please rewrite this with the object oriented matplotlib API for style consistency with the other tutorials, and to promote the use of modern matplotlib syntax (see <a href="https://hoomd-blue.readthedocs.io/en/v4.7.0/tutorial/00-Introducing-HOOMD-blue/07-Analyzing-Trajectories.html" target="_blank">https://hoomd-blue.readthedocs.io/en/v4.7.0/tutorial/00-Introducing-HOOMD-blue/07-Analyzing-Trajectories.html</a> for an example)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>